data/epub.css

```diff
@@ -7,14 +7,19 @@ h3 { text-align: left; }
 h4 { text-align: left; }
 h5 { text-align: left; }
 h6 { text-align: left; }
-/* For title, author, and date on the cover page */
-h1.title { }
+/* For (sub)title, author, date, and publisher on the cover page */
+h1.main { }
```

Comment:
If you look at the default template you'll see it has
```
$for(title)$
$if(title.text)$
  <h1 class="$title.type$">$title.text$</h1>
$else$
  <h1 class="title">$title$</h1>
$endif$
$endfor$
```
Thus, if they use a structured title (e.g. `title { text = ___, type = ___ }`) then "main" will be used (or maybe something else, depending on the value of "type").  If they just specify `title` as an unstructured field, `title` will be used.

Perhaps we could improve this by changing the first case to
```
  <h1 class="title $title.type$">$title.text$</h1>
```
so that the `title` class is consistently applied.
Anyway, I'd suggest we make that change to the template, but also leave out `main` from the default template.
(I'm actually not quite sure what motivated these structured titles -- would need to dig through git history.)